### DISCUSSION TITLE
GPU detection

### DISCUSSION THREAD

**Discussion by mhochst** [talmolab/sleap#1180] [Help!]
hi there, i started using SLEAP recently still new to the program, but i was attempting to have my GPU detected for assisted labeling and for some reason the gpu is not being detected, i have the correct cudakit and cudnn installed as i just completed that and the only thing i get back in the terminal is:
![image](https://user-images.githubusercontent.com/122642039/219475528-99805ea7-0a9e-4ab1-9bf5-b03599b79df2.png)

any advice is greatly appreciated

**Comment by roomrys** [comment]
Hi @mhochst,

Can you try [updating your GPU driver](https://www.nvidia.com/download/index.aspx?lang=en-us)?

This is because sometimes cudatoolkit and cudnn aren't detected with the OS default GPU driver.

If that doesn't work, let us know which GPU you're on and we'll go from there.

Thanks,
Liezl
